src/js/components/Layout/menu/MenuSubsection.jsx

```diff
@@ -1,30 +1,47 @@
-import React from 'react';
+import React, { useRef, useState } from 'react';
 
 import _ from 'lodash';
 import PropTypes from 'prop-types';
 import { RiArrowDropDownLine } from 'react-icons/ri';
 
-const DropdownMenu = ({ section, active }) => (
-  <li className={`nav-item dropdown d-none d-md-flex justify-content-center align-items-center ${active && 'active-section'}`} >
-    <a className="nav-link dropdown-toggle" href="#" id="navbarDropdown" aria-haspopup="true" aria-expanded="false">
-      {section.label}
-    </a>
-    <div className={`dropdown-menu dropdown-menu-wrapper ${section.label === 'Reporting' && 'dropdown-menu-right'}`} aria-labelledby="navbarDropdown">
-      <div className="dropdown-menu-content dropdown-menu-subsections">
-        {_.map(section.subsections, (subsection, subsectionKey) => (
-          <div className="padding-8" key={subsectionKey}>
-            {subsection.label && <span className="subsection-section-title">{subsection.label}</span>}
-            {_.map(subsection.menuItems, (menuItem, menuItemKey) => (
-              <a className="dropdown-item" key={menuItemKey} href={menuItem.href} target={menuItem.target}>
-                {menuItem.label}
-              </a>
-            ))}
-          </div>
-        ))}
+
+const DropdownMenu = ({ section, active }) => {
+  const dropdownRef = useRef(null);
+  const [droppedDown, setDroppedDown] = useState(false);
```

Comment:
I would call this state `[isOpen, setIsOpen] ` instead of `[droppedDown, setDroppedDown]`